src/exception/repo/program.ts

```diff
@@ -43,11 +43,14 @@ const ProgramExceptionModel =
 const programExceptionRepository = {
   async save(exception: ProgramException): Promise<ProgramException> {
     L.debug(`Creating new program exception with: ${JSON.stringify(exception)}`);
+
+    const { exceptions } = exception;
+    const update = { $push: { exceptions } };
```

Comment:
Makes uploading exception records additive. 
So if an exception is already uploaded and you upload a new one, both are stored. 
Currently the app replaces what's in the DB with the new upload.

Reply:
Replacing the exceptions with the content of the uploaded TSV is the agreed upon behaviour. Without this overwrite we would need to add functionality to remove specific records.

Reply:
Reverted